pentasharp/ViewModel/TaxiReservation/UpdateReservationViewModel.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace pentasharp.ViewModel.TaxiReservation
+{
+    /// <summary>
+    /// Represents the view model used to update an existing taxi reservation.
+    /// </summary>
+    public class UpdateReservationViewModel
```

Comment:
why was this not moved to TaxiRequests, and since we offer bus reservation as well if this model is only used for taxis we should rename this to something like UpdateTaxiReservation